public/Aframe/samples/number_lock.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>CoronaEscape</title>
+        <meta name="description" content="CoronaEscape" />
+        <script src="https://aframe.io/releases/1.1.0/aframe.min.js"></script>
+        <script src="aframe-super-keyboard.min.js"></script>
+        <script src="keypad.js"></script>
+    </head>
+    <body>
+        <a-scene
+            cursor="rayOrigin:mouse"
+            background="color: #ECECEC"
+            raycaster="objects: .link"
+        >
+            <a-box
+                static-body="shape: auto"
+                position="0 -0.52 0"
+                rotation="90 0 0"
+                width="20"
+                height="10"
+                depth="1"
+                floor
+            ></a-box>
+            <a-box
```

Comment:
All the boxes are not needed right? Can we get rid of them?

Reply:
Removed